contracts/src/blackjack.cpp

```diff
@@ -331,7 +331,7 @@ void blackjack::on_random(uint64_t ses_id, checksum256 rand) {
             }
             send_game_message(std::vector<param_t>{player_card.get_value()});
             update_state(state_itr, game_state::require_play);
-            require_action(action::play);
+            require_action(action::play, true);
```

Comment:
what if player hits and loses 

and then he wants to double down with splitted cards?